scripts/mc_add_observation.py

```diff
@@ -3,31 +3,35 @@
 # Copyright 2017 the HERA Collaboration
 # Licensed under the 2-clause BSD license.
 
-import argparse,pyuvdata,aipy
+import argparse, aipy
 from hera_mc.observations import Observation
 from hera_mc import mc
-from astropy.time import Time,TimeDelta
+from astropy.time import Time, TimeDelta
+
 a = mc.get_mc_argument_parser()
 a.description = """Read the obsid from a file and create a record in M&C."""
-a.add_argument('files',metavar='file',type=str,nargs='*',default=[],
-        help='*.uv files to extract')
+a.add_argument('files', metavar='file', type=str, nargs='*', default=[],
+               help='*.uv files to extract')
 args = a.parse_args()
 db = mc.connect_to_mc_db(args)
 
 
 for uvfile in args.files:
-    #we need aipy to read the stopt and duration because everything is aweful
-    #but first tell AIPY we're allowed to read these variables
+    # we need aipy to read the stopt and duration because everything is awful
+    # but first tell AIPY we're allowed to read these variables
```

Comment:
The `itemtable` modifications aren't needed anymore since we patched AIPY. That being said, it is strange to me that `startt` doesn't need to be defined. And `duration` isn't used here.